pilot/pkg/serviceregistry/kube/controller/endpoint_builder_test.go

```diff
@@ -286,13 +287,34 @@ func TestNewEndpointBuilderFromMetadataTopologyLabels(t *testing.T) {
 				},
 				opts: Options{ClusterID: c.ctl.cluster},
 			}
-			eb := cc.NewEndpointBuilderFromMetadata(c.proxy)
+			eb := newEndpointBuilderFromMetadata(cc, c.proxy)
```

Comment:
is this test doing anything? it seems we are testing newEndpointBuilderFromMetadata but that function is now deadcode

Reply:
Yeah, i just realized it

Reply:
Will fix